roles/vault/tasks/bootstrap.yml

```diff
@@ -25,7 +25,7 @@
   changed_when: no
   when: "'Vault is already initialized' not in vault_init_output.stdout"
   set_fact:
-    vault_token: "{{ (vault_init_output.stdout|from_json)['root_token'] }}"
+    vault_root_token: "{{ (vault_init_output.stdout|from_json)['root_token'] }}"
```

Comment:
This was `vault_token` because, IIRC, ansible's `set_fact` won't overwrite an already existing variable. So if `vault_root_token` is defined in `security.yml` then the new root token will never be used.